Engine/Results/BacktestingResultHandler.cs

```diff
@@ -54,6 +55,9 @@ public class BacktestingResultHandler : BaseResultsHandler, IResultHandler
         private bool _processingFinalPacket;
         private readonly HashSet<string> _chartSeriesExceededDataPoints;
 
+        protected decimal _closingPortfolioValue;
+        protected DateTime _previousTime;
```

Comment:
We could move these into the `BaseResultsHandler`? Maybe to be more precise, could rename to `_previousUtcSampleTime`/`_dailyPortfolioValue`

nit - `using QuantConnect.Lean.Engine.DataFeeds;` not required